onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -485,6 +489,12 @@ def inc_dataloader():
         model = load_model_with_shape_infer(Path(model_input))  # use smooth quant model for calibration
 
     with tempfile.TemporaryDirectory(prefix="ort.quant.") as quant_tmp_dir:
+        if isinstance(model_input, onnx.ModelProto):
```

Comment:
How about Line 482: does ORTSmoothQuant function could handle ModelProto input?

Reply:
Yes I've checked that `ORTSmoothQuant` supports ModelProto

Reply:
- https://github.com/intel/neural-compressor/blob/d80ab264d6d3995f95d9c562a6ef91ce239b322a/neural_compressor/adaptor/ox_utils/smooth_quant.py#L120
- https://github.com/intel/neural-compressor/blob/d80ab264d6d3995f95d9c562a6ef91ce239b322a/neural_compressor/model/onnx_model.py#L42